Tutorials/NetKetSymmetryInterface.ipynb

```diff
@@ -0,0 +1,689 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install --upgrade "git+https://github.com/netket/netket"
</pre><p>I think this shouldn't be part of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1048/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Oops, I will definitely remove that